src/dripper/Procaptcha.ts

```diff
@@ -0,0 +1,58 @@
+import { getPair } from "@prosopo/contract";
+import { getServerConfig, ProsopoServer } from "@prosopo/server";
+import { ProcaptchaOutput, ProcaptchaOutputSchema } from "@prosopo/types";
+
+import { config } from "src/config";
+
+import { logger } from "../logger";
+
+export class Procaptcha {
+  constructor(
+    private secret: string = config.Get("PROSOPO_SITE_KEY"),
+    private contractAddress: string | undefined = config.Get("PROSOPO_CONTRACT_ADDRESS"),
+    private readonly endpoint: string | undefined = config.Get("PROSOPO_SUBSTRATE_ENDPOINT"),
+    private readonly maxVerifiedTime: number = Number(config.Get("PROCAPTCHA_MAX_VERIFIED_TIME")),
+  ) {
+    if (!this.maxVerifiedTime) {
+      this.maxVerifiedTime = 60000;
+    }
+    if (!this.secret) {
+      throw new Error(`⭕ Procaptcha is not configured. Check the PROSOPO_SITE_KEY variable.`);
+    }
+  }
+
+  getValidator() {
+    console.log("Procaptcha secret is ", this.secret);
```

Comment:
Forgotten `console.log`? :)
Also, a bunch of `console.log`'s down below

Reply:
Yeah, totally. I'll remove all `console.log`'s :)